app/controllers/process_plates_controller.rb

```diff
@@ -41,4 +55,21 @@ def create
       format.html { redirect_to(new_process_plate_path) }
     end
   end
+
+  private
+
+  # Returns a list of unique barcodes by removing blanks and duplicates
+  def sanitize_barcodes(barcodes)
+    sanitized_barcodes = [barcodes].map do |s|
+      s.split(/\s/).reject(&:blank?)
+    end
+    sanitized_barcodes.flatten.uniq
+  end
```

Comment:
Other option could be:
barcodes.split(/\s+/).reject(&:blank?).uniq

Reply:
Thanks, I wanted to ask how to test this private method in the controller too.

Reply:
We cannot access private methods in tests to my knowledge. If we want to test it specifically we could move it to the helpers module or make it public. 
If is not very important this function (not being called in other places), the test from the public method can test this functionality.